webapp/app/helper/plausible_helper.py

```diff
@@ -0,0 +1,49 @@
+from flask_babel import lazy_gettext as _l
+
+
+def get_plausible_data(step_name, config):
```

Comment:
Parameter "config" maybe rename to "domain", since we are passing not the config but only that plausible domain value